backend/.env.example

```
@@ -0,0 +1,20 @@
+NODE_ENV=development
+
+# Database configuration
+DB_HOST=postgres
+DB_PORT=5432
+DB_USER=postgres
+DB_PASSWORD=postgres
+DB_NAME=personal_assistant
+
+# Authentication
+JWT_SECRET=change-me
+JWT_EXPIRE=7d
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The JWT_EXPIRE key should go before the JWT_SECRET key

(UnorderedKey)